.github/workflows/ci.yaml

```diff
@@ -1,45 +1,45 @@
-# yamllint disable rule:line-length
-# yamllint disable rule:braces
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow
 
 name: CI
 
 on:
-  pull_request:
+  pull_request: null
   push:
     branches:
-      - master
       - main
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+    
 env:
-  COMPOSER_ROOT_VERSION: 0.1.x-dev
+  COMPOSER_ROOT_VERSION: '0.1.x-dev'
 
 jobs:
   tests:
+    name: Test with PHP ${{ matrix.php-version }}
     runs-on: ubuntu-latest
     strategy:
       matrix:
         php-version: ['7.4', '8.0', '8.1', '8.2', '8.3']
-        coverage: ['pcov']
```

Comment:
A single value does not make a matrix.

Reply:
I reuse this workflow elsewhere, where I could use `xdebug`. And I don't want to maintain several vastly different workflows, please.